06-building-a-program-start-to-finish/writing-tests/output/invertedIndex-spec.js

```diff
@@ -0,0 +1,205 @@
+const values = require('object.values');  //  shim Object.values
```

Comment:
If you declare the variables well enough you'll not need these comments in your tests.